### ISSUE TITLE
fly command shows just error: forbidden message if not logged in

### ISSUE THREAD

**Issue by tahsinrahman** [concourse/concourse#2134]
# Feature Request

## What challenge are you facing?

`fly` command shows `error: forbidden` message if not logged in v3.10.0. Whereas in previous versions, it showed a message like, user is not logged in, please enter `fly -t <target> login` to login - which was more verbose and self-explanatory.

## A Modest Proposal

It would be great if it showed more elaborate error message like previous versions.

**Comment by darshandeshmukh11** [comment]
I am also facing this issue, I made few changes in my concourse pipeline yml file and I am blocked with below error to set the pipeline 
**I am already logged in to my concourse instance**
```
$ fly -t <my-concourse> set-pipeline -p build-sp
ringboot -c concourse-config.yml
error: forbidden
```

**Comment by shashwathi** [comment]
@darshandeshmukh11 @tahsinrahman  

For `fly` to prompt you for `login`, you need to be unauthorized(logged out). `error: forbidden` is displayed if you are logged in with a different team.

**Comment by mhuangpivotal** [comment]
@tahsinrahman This was a regression and we made a fix for the next release (after v3.11.0). Thanks for reporting!
